packages/xo-web/src/common/render-xo-item.js

```diff
@@ -47,6 +47,7 @@ XoItem.propTypes = {
 const XO_ITEM_PROP_TYPES = {
   ...COMMON_PROP_TYPES,
   id: PropTypes.string.isRequired,
+  resourceSet: PropTypes.object,
```

Comment:
IMHO, i think it's preferable to create a new component `SrResourceSetItem`  who fetch the SR from an id passed in its parameter. This will make the resource set 's UI more dynamic.

Reply:
@pdonias  what do you think about that?

Reply:
Agreed!